src/boot/efi/boot.c

```diff
@@ -36,6 +36,7 @@ enum loader_type {
 typedef struct {
         CHAR16 *id; /* The unique identifier for this entry */
         CHAR16 *title_show;
+        UINTN title_show_len;
```

Comment:
why this?

Reply:
I felt like recomputing StrLen(title_show) every time we do the line formatting loop on console mode change was a bad idea.

Reply:
we do much worse. strlen() is ridiculously fast. it's kinda what modern cpus are optimized for. 

it's fine to cache within a function body, but in the structure here, i am not convinced